### ISSUE TITLE
fastq file error in unmapped reads.

### ISSUE THREAD

**Issue by CS791** [FelixKrueger/Bismark#270]
I am using a perl program for Transposable Element analysis, which required unmapped fastq reads from Bismark output. My original raw data was paired end and I used the following command line for Bismark mapping:

**bismark_v0.21.0/bismark ~/Bismark/Genome -bowtie2 --ambiguous --non_directional -unmapped -R 10 score_min L,0,0.6 -N 1 -1 Epiril_22C_R1_L1/Epiril368_22_C_R1_L001_R1_val_1.fq.gz -2 Epiril_22C_R1_L1/Epiril368_22_C_R1_L001_R2_val_1.fq.gz -o New_Output/Epiril368_22C_Rep1_L1.bam**

The unmapped fastq file generated from above command showing an error as you can see the header of one of the file:

/epiTEome/Analysis/Unmapped_reads/Epiril368_4_C_R1_L001_unmapped.fastq 

> ```
> @K00230:34:HMJJFBBXX:1:1101:2869:1226_1:N:0:CAAAANNTAGATAAGGAAAAAAAAAAAATTATTTTAAGTTGTTATNATTTATTAGATATTAATTAATAAANATTATGNANATAAGATATNAGTTNGTTAATAGAGAATAAGAATATTATAAATTTAATGATTAATTAATAANATGAAAAAAGGGA
> +
> #AA-FJJJFFJFJJJJJJJJFJJJJJAJJJFAF-AFFJJ#JAJJJJ<FAF<JFJJJJFJJFAJJ#JJJFAJ#F#J<F-<F-J-#<--<#<<FJJAF<FFJF-JJ<JJ<JFFJ-AJJ----<--<-<FJA-AFAJJ#FA-7AA<<F7--A
> @K00230:34:HMJJFBBXX:1:1101:3031:1226_1:N:0:CAAAANNAATGGTTCGTATCTGAGTTTTTTTATCTGAATAATTTTNTATTTTTTTTTTGAAGAAAAAAATNATTAAANTNTTGATATTTNGAGANATTTTTAATTCGAATTTTTTTTTTTGTAACCTTTTAGTTTTTCGATNTTTTTTTTTTCTTT
> +
> #AAAFJJJFJJJJJJJJJJJFFJJJJJJ<JJFJJJJJJJ#JJJJFJJJJJJJFFFJJJJJJFJF#JJJJFJ#J#JJJFJAJJJ#-<AA#7FJJJJJFJF-AJ7FJJJJJJJJJJJF7A-7-7F-A-FJ<JJ-7FF#FJJJJJF<FJ-7-F
> @K00230:34:HMJJFBBXX:1:1101:3417:1226_1:N:0:CAAAANNGAAGTTTGGTTATTGTTTTGGAGTCGAATATGATTTGANGTTATGTGTATGATTGAGTATAAGNATTTAANTNGCAACCGGANTTTANAAGTTTAAGTAGTGTGTTTTTGTTAGAATATACAAAGTTAAAGATTNATATGGATTTTGGT
> +
> #AAFFJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJJJJ#JJJJJJFJFJJJJJJJJJJJJJJJ#JJJJJJ#J#JJFJJJJJF#JJJJ#JJJ<JJJJJFJJJJAJFJAJJJJFJJJJJJJJJJJJFJJJJJJJJJ#JJJJJJF7FFJJJJ
> @K00230:34:HMJJFBBXX:1:1101:3945:1226_1:N:0:CAAAANNATATTTTGTTAAAGATTTTTTTTTGTTTTTGTTTTTTTNTTAAAGAATATAATTTTAATAAATNTGGATANANATAGTGAATNTGTTNTGGTTGAGTTAAAATTAGAATTATATTGTAAATTTTGTGTATTTAGNTAAATTTTAAAAT

```
Any suggestion to troubleshoot this issue. 

Thank you!
Regards

CS791

**Comment by FelixKrueger** [comment]
You just need to update to the latest version, this has already been fixed for v0.22.0:

```
fixed missing end-of-line character for unmapped or ambiguous FastQ sequences in paired-end FastQ mode
```

**Comment by FelixKrueger** [comment]
I assume updating Bismark has worked for you?
